WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -37,20 +37,25 @@ public static InputRoundSignaturePair CreateInputRoundSignaturePair(Key? key = n
 			}
 		}
 
-		public static IEnumerable<InputRoundSignaturePair> CreateInputRoundSignaturePairs(int count, uint256? roundHash = null)
```

Comment:
Arraying in this file was necessary, because very strange things were happening with IEnumerables.